src/lib/backend-api.ts

```diff
@@ -492,7 +492,32 @@ export const withCurrency = (backendApiUrl: string, currency: Currency): Currenc
     return response.data
   }
 
-  const listPolicies = async (token: string, limit : number, nextPageToken?:string):Promise<ListPoliciesResponse> => {
+  const listTransfers = async (token: string, walletId: string, limit: number, nextPageToken?: string): Promise<ListTransfersBackendResponse> => {
+    const options = {
+      method: 'GET',
+      headers: {
+        Authorization: token
+      }
+    }
+    const nextPageParam = nextPageToken ? `&nextPageToken=${nextPageToken}` : ''
+    const queryString = `?limit=${limit}${nextPageParam}`
+
+    const response = await request(`${backendApiUrl}/${currency}/wallet/${walletId}/transfers${queryString}`, options)
+    return response.data
```

Comment:
a to nie lepiej zamiast kopiować dodać do istniejącej metody opcjonalny parametr walletId?

Reply:
Chyba nie da sie tak zrobic bo te ogolne transfery sa chain agnostic i nie wiedzialy by z jakiego chaina brac ten wallet